workload/src/main/java/com/google/finapp/WorkloadMain.java

```diff
@@ -0,0 +1,154 @@
+  private static final String DEFAULT_TRANSFER_AMOUNT = "20";
+  private static final Logger logger = Logger.getLogger(WorkloadMain.class.getName());
+
+  private static class WorkloadGenerator {
+    private final ManagedChannel channel;
+    private final List<ByteString> ids = new ArrayList<>();
+
+    WorkloadGenerator(ManagedChannel channel) {
+      this.channel = channel;
+    }
+
+    void seedData(int numAccounts) {
+      int numFailedCreateAccounts = 0;
+      for (int i = 0; i < numAccounts; i++) {
+        try {
+          ByteString response =
+              WorkloadClient.getWorkloadClient(channel)
+                  .createAccount(
+                      DEFAULT_ACCOUNT_BALANCE,
+                      CreateAccountRequest.Type.CHECKING,
+                      CreateAccountRequest.Status.ACTIVE);
+          ids.add(response);
+        } catch (StatusRuntimeException e) {
+          numFailedCreateAccounts++;
+          logger.log(
+              Level.WARNING,
+              String.format("CreateAccount failed. Total count: %d", numFailedCreateAccounts));
+        }
+      }
+    }
+
+    void startSteadyLoad() {
+      Random random = new Random();
+      int numIds = ids.size();
+      if (numIds == 0) {
+        throw new IllegalStateException("No accounts were created successfully.");
+      }
+      while (true) {
+        ByteString fromId = ids.get(random.nextInt(numIds));
+        ByteString toId = ids.get(random.nextInt(numIds));
+        if (fromId.equals(toId)) {
+          continue;
+        }
+        WorkloadClient.getWorkloadClient(channel)
+            .moveAccountBalance(fromId, toId, DEFAULT_TRANSFER_AMOUNT);
+      }
+    }
+  }
+
+  public static void main(String[] args) {
+    CommandLine cmd = parseArgs(args);
+    String addressName = cmd.getOptionValue("a");
+    int port;
+    int numAccounts;
+    try {
+      port = ((Number) cmd.getParsedOptionValue("p")).intValue();
+      numAccounts = ((Number) cmd.getParsedOptionValue("n")).intValue();
+    } catch (ParseException e) {
+      throw new IllegalArgumentException("Input value cannot be parsed.", e);
+    }
+    ManagedChannel channel =
+        ManagedChannelBuilder.forAddress(addressName, port).usePlaintext().build();
+    WorkloadGenerator workloadGenerator = new WorkloadGenerator(channel);
+    workloadGenerator.seedData(numAccounts);
+    workloadGenerator.startSteadyLoad();
+  }
+
+  private static CommandLine parseArgs(String[] args) {
+    Options options = new Options();
+
+    options.addOption(
+        Option.builder("a")
+            .longOpt("address-name")
+            .desc("server address name")
+            .required(true)
+            .type(String.class)
+            .hasArg()
+            .build());
+
+    options.addOption(
+        Option.builder("p")
+            .longOpt("port")
+            .desc("server port")
+            .required(true)
+            .type(Number.class)
```

Comment:
`Integer.class` is not allowed here, and `Number.class` is used instead. See: https://stackoverflow.com/questions/5585634/apache-commons-cli-option-type-and-default-value